draft-packages/likert-scale-legacy/KaizenDraft/LikertScaleLegacy/LikertScaleLegacy.tsx

```diff
@@ -158,34 +157,34 @@ export const LikertScaleLegacy = ({
                 [styles.unselected]:
                   selectedItem && selectedItem.value < item.value,
               })}
-              onClick={(): void => handleRadioClick(item)}
               key={item.value}
               data-automation-id={
                 automationId && `${automationId}-item-${item.value}`
               }
+              onClick={(): void => handleRadioClick(item)}
               onMouseEnter={(): void => handleRadioMouseEnter(item)}
-              onMouseLeave={handleRadioMouseLeave}
+              onMouseLeave={(): void => setHoveredItem(null)}
+              onKeyDown={(event): void => handleKeyDown(event, item)}
+              onFocus={(): void => setHoveredItem(item)}
+              onBlur={(): void => setHoveredItem(null)}
               role="radio"
               aria-label={item.label}
-              aria-checked={
-                selectedItem ? item.value === selectedItem.value : false
-              }
+              aria-checked={isSelectedItem}
               aria-posinset={item.value}
               aria-setsize={5}
               tabIndex={tabIndex}
               ref={itemRef && itemRef.ref}
-              onKeyDown={(event): void => handleKeyDown(event, item)}
             >
               <div
                 className={classnames(
                   styles.likertItemFill,
                   styles[`field${item.value}`],
                   {
-                    [styles.pop]:
-                      selectedItem && item.value === selectedItem.value,
+                    [styles.pop]: isSelectedItem,
                   }
                 )}
               />
```

Comment:
This is a self-closing `div`...? If it's not too much hassle, I think you should open this `div` and place the `Icon` inside it to avoid having to mess with `absolute` CSS

Reply:
I did that at some point and tried to use `flex` to put the icon in the right position, had some weird side effects on the sizing of the likert scale :( This is all very complex and fragile.